subprojects/internal-integ-testing/src/main/groovy/org/gradle/integtests/fixtures/executer/OutputScrapingExecutionResult.java

```diff
@@ -58,6 +58,9 @@
 
     private static final Pattern BUILD_RESULT_PATTERN = Pattern.compile("BUILD (SUCCESSFUL|FAILED)( \\d+[smh])+");
 
+    // JUnit 5's test case name contains parentheses which might break test assertion, e.g. testMethod() PASSED -> testMethod PASSED
```

Comment:
❌ I think we should not do this here, but in the Junit multiversion base class. Otherwise we risk matching other things with this pattern in the future. (We already had similar issues when dependency resolution graphs were wrongly interpreted as exception stack traces :)

Reply:
Thanks. Now they're in `JUnitMultiVersionIntegrationSpec`.